### PR TITLE
Redshift lloyds

### PR DESCRIPTION
**Pull request by shulkaolka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,12 +98,114 @@ For example, the incoming message contains the following body:
...
+   * number 
+   * boolean 
+   
+For example, for sql expression `SELECT * FROM tableName WHERE column1 = 'text' AND column2 = 15` you need to use following template:
+`SELECT * FROM tableName WHERE column1 = @column1:string AND column2 = @column2:number` and put values into generated metadata.
+
+![image](https://user-images.githubusercontent.com/16806832/53731432-635a2200-3e83-11e9-9a4e-0fc26aeeb001.png)
+
+### Input metadata
+Input metadata is generated from `SQL Query` configuration field if this field contains at list one defined value.
```
Comment (kirill-levitskiy-elasticio) [inline README.md]: Misprint: 
Input metadata is generated from `SQL Query` configuration field if this field contains at *least* one defined value.

File: README.md [hunk 2]
```diff
@@ -79,12 +98,114 @@ For example, the incoming message contains the following body:
...
+## Execute Sql Injection
+
+**Expert mode**. You can execute Sql Injection in this action.
+You can not use prepare statement there, for this purpose use **General Sql Query Action**.
+
+### Input metadata
+
+Input metadata contains one field `Sql Injection string`. You can put there SQL query, SQL script or set of SQL queries from the previous step.
+You can put only one SQL query or several queries with delimiter `;`.
+All queries are executed in one transaction. All changes will rollback, if something wrong with one of execution.
```
Comment (kirill-levitskiy-elasticio) [inline README.md]: Misprint:
All queries are executed in one transaction. All changes will rollback, if something wrong with one of *executions*.

File: component.json [hunk 3]
```diff
@@ -13,35 +13,58 @@
     }
   },
   "actions": {
-    "insert" : {
-      "title": "INSERT/UPDATE/DELETE",
-      "main": "./lib/actions/insert.js",
+    "insertBulk" : {
+      "title": "INSERT Bulk",
+      "main": "./lib/actions/insertBulk.js",
+      "description": "Insert multiple rows specified in incoming message in defined columns in a single table",
```
Comment (kirill-levitskiy-elasticio) [inline component.json]: "description": "Insert multiple rows specified at the incoming message into defined columns in a single table",

File: component.json [hunk 4]
```diff
@@ -13,35 +13,58 @@
...
-          "label": "SQL Query",
+        "tableName": {
+          "label": "Table Name",
           "required": true,
-          "viewClass": "TextAreaWithNoteView",
-          "placeholder": "INSERT INTO films (code,title,kind) VALUES (${code},${title},${kind})",
-          "note": "You can use properties of message body as <i>${values}</i> in your insert or update or delete"
+          "viewClass": "TextFieldView",
+          "placeholder": "schema.tableName",
+          "note": "Enter the name of the table into which the values will be inserted"
```
Comment (kirill-levitskiy-elasticio) [inline component.json]: "note": "Enter the name of the table where values will be inserted"

File: component.json [hunk 5]
```diff
@@ -13,35 +13,58 @@
...
+        },
+        "columns": {
+          "label": "Columns",
+          "required": true,
+          "viewClass": "TextAreaView",
+          "placeholder": "column1, column2, column3",
+          "note": "Enter the name of the columns into which the values will be inserted"
```
Comment (kirill-levitskiy-elasticio) [inline component.json]: "note": "Enter the name of columns where values will be inserted"

File: component.json [hunk 6]
```diff
@@ -13,35 +13,58 @@
...
+      "main": "./lib/actions/insert.js",
+      "description": "INSERT or UPDATE or DELETE SQL Query",
       "fields": {
         "query": {
           "label": "SQL Query",
           "required": true,
           "viewClass": "TextAreaWithNoteView",
           "placeholder": "INSERT INTO films (code,title,kind) VALUES (${code},${title},${kind})",
-          "note": "You need use properties of message body as <i>${values}</i> in your  insert query"
+          "note": "You can use properties of message body as <i>${values}</i> in your insert or update or delete"
```
Comment (kirill-levitskiy-elasticio) [inline component.json]: "note": "You can use properties of message body as <i>${values}</i> in your insert or update or delete query"

File: package.json [hunk 7]
```diff
@@ -21,23 +21,31 @@
     "elasticio-component"
   ],
   "dependencies": {
-    "elasticio-node": "0.0.5",
-    "elasticio-sailor-nodejs": "2.1.0",
-    "lodash": "^4.6.1",
+    "dotenv": "6.2.0",
+    "elasticio-node": "0.0.9",
+    "elasticio-sailor-nodejs": "^2.3.0",
```
Comment (kirill-levitskiy-elasticio) [inline package.json]: Should we use here `sailor 2.2.3` with flow-control feature?

Comment (shulkaolka) [reply package.json]: New sailor didn't help to resolve the problem with OOM, so I made the decision to leave current sailor

**Pull request merged**